recipes/wcp/icon/v1/a100/environments.yaml

```diff
@@ -14,10 +14,11 @@ nvidia:
   - eccodes@2.25.0%nvhpc +tools +fortran +aec +openmp jp2k=jasper
   - cmake%gcc
   - cuda@12.3%gcc
-  - hdf5%gcc
+  - openblas%gcc
   - hwloc%gcc
-  - netcdf-c%gcc
-  - netcdf-cxx4%gcc
+  - hdf5+szip+fortran%nvhpc
```

Comment:
Do we need the `fortran` interface to `hdf5`? We have observed some bugs with `hdf5` built using nvhpc on Santis - so we prefer to build using gcc without the Fortran interface if it is not required.

Reply:
Same here, the concretizer didn't take it into account so I checked the spec of the rebuild hdf5 and modified the recipe accordingly